code_of_conduct.md

```diff
@@ -0,0 +1,253 @@
+
+**Article I: Member Expectations**
+
+1. Members shall respect the aims and objectives of Texas Lambda Alpha Nu.
+2. Members shall abide by all the rules, policies and procedures found in Texas Lambda Alpha Nu&#39;s Bylaws, initiations rituals, this code of conduct and such additional regulations as may be enacted in accordance with existing rules.
+3. Members will strive to conduct themselves according to Texas Lambda Alpha Nu&#39;s pillars of solidarity, scholarship and ambition.
+4. Members shall be responsible for their own behavior, recognizing that their actions reflect upon themselves, their peers and occasionally others.
+5. Members shall avoid actions that violate or diminish the collegial, academic, personal or professional ethics or the legal and civil rights of peers, the public, university personnel or others who may be affected by their actions.
+6. Members shall respect the dignity of all persons and will not physically, mentally, psychologically abuse or haze any human being nor allow themselves to be physically, mentally, psychologically abused or hazed.
+7. Members shall regard, refer and treat Members and all individuals with civility and respect.
+8. Members shall respect their property and the property of others and will neither abuse nor tolerate the abuse of property.
+9. Members shall pay any and all Chapter dues and assessments required for the Member to remain in good standing with the Chapter.
+10. Members in good standing will make efforts to attend Chapter meetings.
+11. Members who are students shall abide by those academic standards established by LAN and by the college or university as a whole.
+12. In addition to this Code of Conduct and Member Disciplinary Policy, all members who attend colleges and universities are expected to abide by the policies and regulations that govern the duties, responsibilities and conduct explicitly or implicitly described in the university´s publications.
+
+**Chapter II**
+
+**Member Disciplinary Policy**
+
+The Member Disciplinary Policy explains the procedures for reporting violations, what sanctions may be imposed and the basis of which an appeal may be filed. This policy supplements and does not replace other rules of Texas Lambda Alpha Nu that may provide for disciplinary policies, rules or regulations.
+
+**Article I: Reporting Violations**
+
+- Anyone desiring to lodge a complaint, whether formal or informal, shall inform any member of the Standards Board or Vice President of Standards (referred to as VP of Standards).
+- Informal complaints are usually, but not always, solved by talking to the accused about their current behavior and encouraging them to change in the future.
+- Formal complaints will be followed by a documented report, an investigation and a hearing detailed in this document.
+
+**Article II: Distribution**
+
+All members in good standing and prospective members are entitled to receive a copy of the Code of Conduct and Member Disciplinary Policy.
```

Comment:
If that's the intended meaning, "on request" or equivalent ought to be added to this line.